corehq/toggles.py

```diff
@@ -717,3 +717,10 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_ONE_OFF,
     [NAMESPACE_DOMAIN]
 )
+
+SECURE_SESSIONS_CHECKBOX = StaticToggle(
```

Comment:
always love seeing appropriately named and  tagged usage of feature flags!

On Wed, Dec 9, 2015 at 2:27 PM, krsdimagi <notifications@github.com> wrote:

> In corehq/toggles.py
> <https://github.com/dimagi/commcare-hq/pull/9496#discussion_r47138788>:
>
> > @@ -717,3 +717,10 @@ def _commtrackify(domain_name, toggle_is_enabled):
> >      TAG_ONE_OFF,
> >      [NAMESPACE_DOMAIN]
> >  )
> > +
> > +SECURE_SESSIONS_CHECKBOX = StaticToggle(
>
> @emord <https://github.com/emord> the idea now is that we have a variety
> of different options that can be toggled on and off per domain, rather than
> just a single "more secure domain" checkbox. This is both to allow domains
> to be more customizable, and also to allow us to have features exist at
> different levels of payment plan once we decide exactly how we want to
> split them up.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9496/files#r47138788>.
>